CMakeLists.txt

```diff
@@ -572,26 +572,30 @@ if(WIN32)
   add_compile_options(/Zl) # omit default library name in .OBJ
 endif(WIN32)
 
+#--------------------------------
+# Definition directives
+#  - all clr specific compile definitions should be included in this file
+#  - all clr specific feature variable should also be added in this file
+#----------------------------------
+include(clrdefinitions.cmake)
```

Comment:
looks like I have to build the GC pal before the CoreCLR PAL, but I can move the definitions back to their original place.